packages/libexecinfo.rb

```diff
@@ -0,0 +1,29 @@
+require 'package'
+
+class Libexecinfo < Package
+  description 'BSD licensed clone of backtrace facility'
+  homepage 'https://github.com/mikroskeem/libexecinfo'
+  version '1.1'
+  license 'BSD'
+  compatibility 'all'
+  source_url 'https://github.com/mikroskeem/libexecinfo.git'
+  git_hashtag 'd79eee9b042198a841411668368fcad18a3da506'
```

Comment:
```suggestion
  git_hashtag 'version'
```

there is a 1.1 tag here: https://github.com/mikroskeem/libexecinfo/releases/tag/1.1

Reply:
I know.  I used the latest hash since all their tagged releases were basically just rebuilds.

Reply:
It is still best practice to use the version tag of the version we are shipping.

Reply:
OK, you win but I removed the quotes. :)

Reply:
oops i forgot about those good catch